.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "eslint.autoFixOnSave": true
+}
```

Comment:
Don't commit `.vscode` stuff ... these are your own personal settings. Add it to `.gitignore`